gradio/blocks.py

```diff
@@ -504,6 +504,7 @@ def __init__(
         css: str | None = None,
         js: str | None = None,
         head: str | None = None,
+        fill_vertical_space: bool = False,
```

Comment:
What about just renaming this parameter `full_height`. Its less verbose and clearer than `fill_vertical_space` imo

Reply:
full_height is not accurate though. All apps take full height with the footer. This parameter specifically means that top level components, by default, expand to fill the vertical space. Open to other names, cc: @pngwn